rotkehlchen/tests/api/test_pnl_csv.py

```diff
@@ -157,6 +162,34 @@ def test_history_export_download_csv(
         with zipfile.ZipFile(tempzipfile, 'r') as zip_ref:
             zip_ref.extractall(extractdir)
         assert_csv_export_response(response, extractdir, is_download=True)
+    # check if it works with custom assets
+    custom_asset = CustomAsset.initialize(
+        identifier='XYZ',
+        name='custom name',
+        custom_asset_type='custom type',
+    )
+    db_custom_assets = DBCustomAssets(rotkehlchen_api_server_with_exchanges.rest_api.rotkehlchen.data.db)  # noqa: E501
+    db_custom_assets.add_custom_asset(custom_asset)
+    accounting_history_process(
+        rotkehlchen_api_server_with_exchanges.rest_api.rotkehlchen.accountant,
+        start_ts=Timestamp(0),
+        end_ts=Timestamp(1640493376),
+        history_list=[EvmEvent(
+            tx_hash=make_evm_tx_hash(),
```

Comment:
instead of using a evmEvent can you use a normal history event? it works but the typing is not correct since evm event is meant to be used with cryptoassets. In the definition we have asset but to avoid having to resolve every asset that goes in there